test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -25,13 +24,19 @@ public class TrustedTestCert<T> : IDisposable
         public TrustedTestCert(T source,
             Func<T, X509Certificate2> getCert,
             StoreName storeName = StoreName.TrustedPeople,
-            StoreLocation storeLocation = StoreLocation.CurrentUser)
+            StoreLocation storeLocation = StoreLocation.CurrentUser,
+            TimeSpan? maximumValidityPeriod = null)
         {
             Source = source;
             TrustedCert = getCert(source);
 
+            if (!maximumValidityPeriod.HasValue)
+            {
+                maximumValidityPeriod = TimeSpan.FromHours(2);
+            }
+
 #if IS_DESKTOP
-            if (TrustedCert.NotAfter - TrustedCert.NotBefore > TimeSpan.FromHours(2))
+            if (TrustedCert.NotAfter - TrustedCert.NotBefore > maximumValidityPeriod.Value)
             {
                 throw new InvalidOperationException("The cert used is valid for more than two hours.");
```

Comment:
Update the exception message to reflect the actual maximum validity period.

Reply:
Yeah, good call.